homeassistant/components/viam/strings.json

```diff
@@ -0,0 +1,168 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Authenticate with Viam",
+        "description": "Select which credential type to use.",
+        "data": {
+          "credential_type": "Credential Type"
+        }
+      },
+      "auth": {
+        "title": "Authenticate with Viam",
+        "description": "Provide the credentials for communicating with the Viam service.",
+        "data": {
+          "api_id": "API Key ID",
+          "api_key": "API Key Value",
+          "address": "Robot Address",
+          "secret": "Robot Secret"
+        },
+        "data_description": {
+          "address": "Find this under the Code Sample tab in the app.",
+          "secret": "Find this under the Code Sample tab in the app when 'include secret' is enabled."
+        }
+      },
+      "robot": {
+        "data": {
+          "robot": "Select a robot"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "selector": {
+    "credential_type": {
+      "options": {
+        "api-key": "Org API Key",
```

Comment:
```suggestion
        "api-key": "Org API key",
```

Reply:
Can you provide more feedback about this change? Is it subjective or part of a style guide?

Reply:
Text must be in sentence case (it's not written anywhere, but it is a requirement (for at least all new strings))